Formula/picat.rb

```diff
@@ -25,6 +25,8 @@ def install
   end
 
   test do
+    ver = shell_output("#{bin}/picat --version").chomp
+    assert_equal "Picat version 2.6#2", ver
```

Comment:
Please get rid of this new "test".

The goal of tests is to not have to update them with every new upstream release.

You're welcome to add additional testing that tests the software functions correctly, just like the one below.

Reply:
ping @boltomli

Reply:
```suggestion
```